test/nn/hypergraph/test_unigin_layer.py

```diff
@@ -15,7 +15,13 @@ def UniGIN_layer(self):
         self.in_channels = 10
         return UniGINLayer(in_channels=self.in_channels)
 
-    def test_forward(self, UniGIN_layer):
+    @pytest.fixture
+    def UniGIN_layer2(self):
```

Comment:
Avoid mix of under_scores and CamelCase in names.
Python guidelines ask for all lower cases and under_scores in function names, see Pep8. Can you adapt the naming of this function and the others which may have the same problem? Thanks.